atd-vzd/migrations/default/1697144772669_add_ems_trigger_function/up.sql

```diff
@@ -0,0 +1,26 @@
+CREATE OR REPLACE FUNCTION public.ems_incidents_trigger()
+ RETURNS trigger
+ LANGUAGE plpgsql
+AS $function$
+BEGIN
+  update ems__incidents set
+    austin_full_purpose = (
+      select ST_Contains(jurisdiction.geometry, incidents.geometry)
+      from ems__incidents incidents
+      left join atd_jurisdictions jurisdiction on (jurisdiction.jurisdiction_label = 'AUSTIN FULL PURPOSE')
+      where incidents.id = new.id),
+    location_id = (
+      select locations.location_id
+      from ems__incidents incidents
+      join atd_txdot_locations locations on (locations.location_group = 1 and incidents.geometry && locations.shape and ST_Contains(locations.shape, incidents.geometry))
+      where incidents.id = new.id),

```

Comment:
you might want to add `LIMIT 1` in case there were some scenario (crufty polygons?) where the incident intersected with multiple location polygons?

Reply:
that is a good check. I think that since we are limiting to `location_group = 1` here takes care of any overlap, but it is hard to tell from the code what is going on. Group 1 (level 1-4) can overlap with group 2 (level 5 or what we are calling major roads now).

Reply:
@patrickm02L I made the update to include limit, but I think that we can spin off an issue to address the idea of making this always return the same location polygon to associate with the EMS incident. Frank had the idea of using the centroid or something similar.